docs/identity-platform/apple-sso-plugin.md

```diff
@@ -54,6 +54,7 @@ To use the Microsoft Enterprise SSO plug-in for Apple devices:
 
   - `app-site-association.cdn-apple.com`
   - `app-site-association.networking.apple`
+  - `config.edge.skype.com`
```

Comment:
Can you add a note below to clarify why it is needed?

Reply:
Good idea, updated.